runners/console/index.ts

```diff
@@ -215,6 +215,32 @@ export class Console extends Runner {
         return result;
     }
 
+    runDownloadFile(step: Step, command: Command): RunResult {
+        let result = new RunResult();
+        result.returnCode = 0;
+
+        let downloadUrl = command.parameters[0];
+        let installDir = path.join(this.getWorkingDirectory(), "devonfw", "workspaces", "main", command.parameters[1]);
```

Comment:
Use the new path handling to get the working directory
https://github.com/devonfw-forge/tutorial-compiler/pull/100